### PR TITLE
Add /ph rewrite

### PR DESCRIPTION
**Pull request by TheOtterlord**
#### What kind of changes does this PR include?

- Something else!

#### Description

- What does this PR change? Adds a `/ph` rewrite to https://astro-houston-ph.pages.dev/

### CODE REVIEW

**Comment by TheOtterlord** [comment]
Looking at the [preview](https://deploy-preview-573--astro-starlight.netlify.app/ph), I'm not sure if this has worked

File: docs/public/_redirects [hunk 1]
```diff
@@ -1 +1,2 @@
 /:lang/*    /:lang/404/   404
+/ph/ https://astro-houston-ph.pages.dev/ 200
```
Comment (delucis) [inline docs/public/_redirects:2]: I suspect we need the opposite order here. Netlify processes these top to bottom so the `/:lang/*` pattern is matching `/ph/` first.

```suggestion
/ph/        https://astro-houston-ph.pages.dev/   200
/:lang/*    /:lang/404/   404
```

Comment (TheOtterlord) [reply docs/public/_redirects:2]: Ahh, this makes a lot of sense :sweat_smile:

File: docs/public/_redirects [hunk 2]
```diff
@@ -1 +1,2 @@
+/ph/        https://astro-houston-ph.pages.dev/   200
```
Comment (TheOtterlord) [inline docs/public/_redirects:1]: Looks like we need a wildcard to match all routes
```suggestion
/ph/*        https://astro-houston-ph.pages.dev/:path   200
```

**Comment by TheOtterlord** [comment]
`/ph/` worked for the page, but didn't work for subroutes. the wildcard doesn't seem to work for any of them, but does prevent the normal 404 from rendering :thinking:

File: docs/public/_redirects [hunk 3]
```diff
@@ -1 +1,2 @@
+/ph/*        https://astro-houston-ph.pages.dev/:path   200
```
Comment (TheOtterlord) [inline docs/public/_redirects:1]:
```suggestion
/ph/*        https://astro-houston-ph.pages.dev/:splat  200
```

**Comment by TheOtterlord** [comment]
Looks like this works, but I'll need to update the site to use a `/ph` base route

File: docs/public/_redirects [hunk 4]
```diff
@@ -1 +1,2 @@
+/ph/*        https://astro-houston-ph.pages.dev/:splat  200
```
Comment (TheOtterlord) [inline docs/public/_redirects:1]:
```suggestion
/ph/*        https://astro-houston-ph.pages.dev/ph/:splat  200
```

**Comment by TheOtterlord** [comment]
This works now! [preview](https://deploy-preview-573--astro-starlight.netlify.app/ph/)

**Review by delucis — APPROVED** [review]
Great work! LGTM! 

(We can merge this now — no need to wait for next week)

**Pull request merged**